internal/k8s/builder/testdata/multiple-instances.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: api-gateway.consul.hashicorp.com/v1alpha1
+kind: GatewayClassConfig
+metadata:
+  name: test-gateway-class-config
+spec:
+  image:
+    consulAPIGateway: hashicorp/consul-api-gateway:0.2.1
+  serviceType: "ClusterIP"
+  deployment:
+    defaultInstances: 10
```

Comment:
Should we expect an `8` here now?

Reply:
Good point. I took out the logic that's actually checking it since I added the max value to the kube validation, but I'll update the test for consistencies sake.